### PR TITLE
Adds AMR Profile status to periodic meter reads E & G

### PR DESCRIPTION
**Pull request by rudidevries**
Connects to SmartSocietyServices/Protocol-Adapter-DLMS#18
Connects to SmartSocietyServices/Webapps#239

### CODE REVIEW

File: osgp-adapter-domain-smartmetering/src/main/java/com/alliander/osgp/adapter/domain/smartmetering/application/mapping/MonitoringMapper.java [hunk 1]
```diff
@@ -16,12 +16,18 @@
...
-
         mapperFactory.getConverterFactory().registerConverter(new AlarmRegisterConverter());
         mapperFactory.getConverterFactory().registerConverter(new ActualMeterReadsConverter());
         mapperFactory.getConverterFactory().registerConverter(new ActualMeterReadsGasConverter());
         mapperFactory.getConverterFactory().registerConverter(new PeriodicMeterReadsRequestConverter());
         mapperFactory.getConverterFactory().registerConverter(new PeriodicMeterReadsResponseConverter());
         mapperFactory.getConverterFactory().registerConverter(new PeriodicMeterReadsGasResponseConverter());
+
+        mapperFactory.getConverterFactory().registerConverter(AmrProfileStatusCodeConverter.CONVERTER_ID,
+                new AmrProfileStatusCodeConverter());
```
Comment (bvdzwet) [inline osgp-adapter-domain-smartmetering/src/main/java/com/alliander/osgp/adapter/domain/smartmetering/application/mapping/MonitoringMapper.java]: I'm not sure why the registerConverter methods with the first parameter as a String are used, but maybe there's a good reason these are not like the earlier ones.

Comment (rudidevries) [inline osgp-adapter-domain-smartmetering/src/main/java/com/alliander/osgp/adapter/domain/smartmetering/application/mapping/MonitoringMapper.java]: I see now that I have misinterpreted the Orika API a bit. It is possible to use Map directly and leave the converters without the name. I will change this.

File: osgp-domain-core/src/main/java/com/alliander/osgp/domain/core/valueobjects/smartmetering/PeriodicMeterReads.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.alliander.osgp.domain.core.valueobjects.smartmetering;
```
Comment (bvdzwet) [inline osgp-domain-core/src/main/java/com/alliander/osgp/domain/core/valueobjects/smartmetering/PeriodicMeterReads.java]: This file is missing a license header.

File: osgp-domain-core/src/main/java/com/alliander/osgp/domain/core/valueobjects/smartmetering/PeriodicMeterReadsGas.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.alliander.osgp.domain.core.valueobjects.smartmetering;
```
Comment (bvdzwet) [inline osgp-domain-core/src/main/java/com/alliander/osgp/domain/core/valueobjects/smartmetering/PeriodicMeterReadsGas.java]: This file is missing a license header.

**Pull request merged**